test/hmm_test.cpp

```diff
@@ -64,6 +64,12 @@ TEST(ScalarMath, PowerF)
     EXPECT_FLOAT_EQ(HMM_PowerF(-2.0f, 4.1), -17.148376f);
     EXPECT_FLOAT_EQ(HMM_PowerF(2.0f, -2.5), 0.176777f);
     EXPECT_FLOAT_EQ(HMM_PowerF(-2.0f, -2.5), -0.176777f);
+
+TEST(ScalarMath, Lerp)
+{
+    EXPECT_FLOAT_EQ(HMM_Lerp(-2.0f, 0.0f, 2.0f), -2.0f);        
+    EXPECT_FLOAT_EQ(HMM_Lerp(-2.0f, 0.5f, 2.0f), 0.0f);        
+    EXPECT_FLOAT_EQ(HMM_Lerp(-2.0f, 1.0f, 2.0f), 2.0f);
```

Comment:
Or maybe it's just a missing closing brace. In any case, once that's fixed, Travis should be happy.